File: Kotlin/problem006.kt
/*
 * Problem 6 - Project Euler
 * http://projecteuler.net/index.php?section=problems&id=6
 */
fun main(args: Array<String>) {
    val limit = 100;
    val sum = (1 + limit) * limit / 2
    val sumsq = sum * sum
    val sqsum = (1..limit).map { it * it }.sum()
    println(sumsq - sqsum)
}

File: TypeScript/problem006.ts
/*
 * Problem 6 - Project Euler
 * http://projecteuler.net/index.php?section=problems&id=6
 */
{
    let sqsum = 0;
    let sumsq = 0;
    for (let i = 1; i <= 100; i++) {
        sqsum += i * i;
        sumsq += i;
    }
    console.log(sumsq * sumsq - sqsum);
}
